src/script/api.js

```diff
@@ -75,13 +75,24 @@ function api(base, defaultOptions) {
 		};
 	}
 
+	function APICall(method, url, defaultData) {
+		return (data, options) => {
+			const body = Object.assign({}, defaultData, data);
+			body.options = Object.assign(body.options || {},
+				defaultOptions, options);
+			return info.then(() => request(method, url, JSON.stringify(body), {
+				'Content-Type': 'application/json'
+			}));
+		};
+	}
+
 	return Object.assign(info, {
 		convert: indigoCall('POST', 'indigo/convert'),
 		layout: indigoCall('POST', 'indigo/layout'),
-		clean: indigoCall('POST', 'indigo/clean'),
```

Comment:
if we're stripping out indigo entirely we may want to remove the indigoCall function as well.